### PR TITLE
add: added rule page

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
Added rule page

image

### CODE REVIEW

**Review by Muzammil-khan710 — APPROVED** [review]
Code is clean & well maintained. Good to merge

File: pages/rule-page/rule-page.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
+@import url(/pages/color.css);
+@import url(/style.css);@import url(/pages/color.css);
```
Comment (gauravsinhaweb) [inline pages/rule-page/rule-page.css:2]: separate line for indentation

File: pages/rule-page/rule-page.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="rule-page.css">
+    <title>Quartz Quiz</title>
+</head>
+
+<body>
+
+
```
Comment (pradipgharge) [inline pages/rule-page/rule-page.html:14]: remove extra spacing here.

Comment (gauravsinhaweb) [inline pages/rule-page/rule-page.html:14]: remove spacing.

File: pages/rule-page/rule-page.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    <header class="header-nav">
+        <div class="navbar">
+
+            <div class="logo">
+                <h1><i class="fab fa-artstation"></i>Quartz Quiz</h1>
+            </div>
+
+
```
Comment (gauravsinhaweb) [inline pages/rule-page/rule-page.html:22]: remove spacing.

File: pages/rule-page/rule-page.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        </div>
+    </header>
+
+    <main class="ques1">
+
+        <div>
+            <h2 class="title">Rules</h2>
+        </div>
+
+
```
Comment (gauravsinhaweb) [inline pages/rule-page/rule-page.html:32]: remove spacing.

File: pages/rule-page/rule-page.html [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        <div class="ques">
+            <ul class="rule-list">
+                <li>You will be awarded 10 points for correct answer.</li>
+                <li>You can't go to previous again.</li>
+                <li>You can't quit while playing it.</li>
+                <li>Correct answer will be displayed at the end.</li>
+            </ul>
+        </div>
+
+
```
Comment (gauravsinhaweb) [inline pages/rule-page/rule-page.html:42]: remove spacing.

File: pages/rule-page/rule-page.css [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    font-family: 'Roboto Condensed', sans-serif;
+    background: rgb(250, 238, 240);
+}
+
+.header-nav {
+    height: 5rem;
+}
+
+.ques1 {
+    background: var(--white-less2);
```
Comment (pradipgharge) [inline pages/rule-page/rule-page.css:21]: can use a better name for the color.

**Review by gauravsinhaweb — APPROVED** [review]
Good to merge after changes

**Review by pradipgharge — APPROVED** [review]
added two comments, post changes it's good to merge!

**Pull request merged**